cmd/cortex/main.go

```diff
@@ -139,6 +141,9 @@ func main() {
 
 	util_log.InitLogger(&cfg.Server)
 
+	//	Set the Go standard library logger to use the same logger as Cortex
```

Comment:
[nit] Just removing whitespaces.

```suggestion
	// Set the Go standard library logger to use the same logger as Cortex.
```

Reply:
👍